predicators/spot_utils/perception/object_specific_grasp_selection.py

```diff
@@ -65,9 +65,7 @@ def _get_ball_grasp_pixel(
     mask = seg_bb.mask
     pixels_in_mask = np.where(mask)
     pixel = (pixels_in_mask[1][-1], pixels_in_mask[0][-1])
-    # Force a forward top-down grasp.
-    rot_quat = math_helpers.Quat.from_pitch(np.pi / 2)
-    return pixel, rot_quat
+    return pixel, None
```

Comment:
just double checking that this change is intentional @NishanthJKumar

Reply:
intentional!